bin/generate-typescript.js

```diff
@@ -0,0 +1,5 @@
+const execa = require("execa");
+
+console.log("⌛  Generating schema.d.ts …");
+
+execa("npx", ["graphql-codegen"]);
```

Comment:
I wonder if it makes sense to create a JS file for it, if all it does is to run `graphql-codegen`? I think I'd instead update the script in package.json to

```
     "generate-typescript": "graphql-codegen",
```